source/voice_control/troubleshooting.markdown

```
@@ -87,6 +87,25 @@ The example below shows common pitfalls when enquiring about the weather. While
+The voice response is generated in Home Assistant by one of our supported text-to-speech (or {% term TTS %}) engines.
+The voice assistant device then fetches the audio file from Home Assistant and plays it back.
+
+For this fetching process to work, Home Assistant must communicate its own URL to the device.
+If you have a complex network setup, or if you changed this setting in the past, the URL communicated could be wrong.
+
+To fix the URL, follow these steps:
+
+1. In {% my profile title="your user profile" %}, enable **Advanced Mode**.
+2. Go to {% my network title="**Settings** > **System** > **Network**" %}.
+3. Change your Local Network Home Assistant URL to a URL that can be reached locally and that points to Home Assistant
+  - For most users, the **Automatic** option works and is recommended.
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)